saleor/graphql/account/types.py

```diff
@@ -281,7 +282,9 @@ class User(CountableDjangoObjectType):
         "saleor.graphql.payment.types.PaymentSource",
         description="List of stored payment sources.",
     )
-    wishlist = PrefetchingConnectionField(WishlistItem, description="User's wishlist.")
+    # wishlist = PrefetchingConnectionField(
+    #   WishlistItem, description="User's wishlist."
+    # )
```

Comment:
Without any comments, it will be hard to understand why it's commented out. TBH I would just remove it.